oasislmf/manager.py

```diff
@@ -417,6 +418,67 @@ def generate_oasis_files(
             exposure_fp, _keys_fp, exposure_profile=exposure_profile
         )
 
+        # Fill summary of exposure data dictionary
+        def fill_exposure_summary(
```

Comment:
This method is clearly not a top level method - it needs to be moved out of `manager.py` into a separate module, probably a module named `summaries.py` in `model_preparation`. @sambles I think your `sum_inputs.py` module is also better renamed as `summaries.py`, and should contain all methods and attributes related to the summary stuff.

Also, the method needs renaming - `get_exposure_summary` is a better method name. In general, for simplicity, any method that is producing and returning data (using `return`, not `yield`) should have a `get` name prefix; any method that produces data but generates it using `yield` should have a `generate` name prefix.